app/code/community/Bolt/Boltpay/Model/Order.php

```diff
@@ -966,6 +966,8 @@ public function removePreAuthOrder($order) {
             if ($this->boltHelper()->getExtraConfig('keepPreAuthOrders')) {return;}
             $previousStoreId = Mage::app()->getStore()->getId();
             Mage::app()->setCurrentStore(Mage_Core_Model_App::ADMIN_STORE_ID);
+            $adminEntry = Mage::getModel('sales/order_grid')->load($order->getId());
```

Comment:
can you explain what this part is?

Reply:
@nirajjayantbolt 
In Magento, deleting orders is not enough to prevent them from being displayed in the admin.   There is a separate table for this exclusively for presentation of the orders.  In some databases, constraints will keep this and the orders table in sync.  In other cases, the constraints are not enabled and this is needed to remove the order from being listed in the admin.  This code is for that case.

Reply:
Oh okay got it thanks!